Code/Framework/AzToolsFramework/Tests/EditorTransformComponentSelectionTests.cpp

```diff
@@ -261,11 +261,16 @@ namespace UnitTest
         AZ::EntityId m_entityId1;
         AZ::EntityId m_entityId2;
         AZ::EntityId m_entityId3;
-        AZ::Vector3 m_entity1WorldTranslation = AZ::Vector3(5.0f, 15.0f, 10.0f);
-        AZ::Vector3 m_entity2WorldTranslation = AZ::Vector3(5.0f, 14.0f, 10.0f);
-        AZ::Vector3 m_entity3WorldTranslation = AZ::Vector3(5.0f, 16.0f, 10.0f);
+
+        static const AZ::Vector3 Entity1WorldTranslation;
+        static const AZ::Vector3 Entity2WorldTranslation;
+        static const AZ::Vector3 Entity3WorldTranslation;
     };
 
+    const AZ::Vector3 EditorTransformComponentSelectionViewportPickingFixture::Entity1WorldTranslation = AZ::Vector3(5.0f, 15.0f, 10.0f);
```

Comment:
Suggestion: these can be declared inline and initialized in place, i.e. `inline static const AZ::Vector3 Entity1WorldTranslation = AZ::Vector3(5.0f, 15.0f, 10.0f);`.

Reply:
The reason I didn't do this is I got a compiler error on linux when going through AR before, but it might be because I was missing `inline`, I'll try and update and see if that works, thanks :+1: